.github/workflows/webonary-cloud-api-pr.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         run: cd webonary-cloud-api/lambda; npm ci
 
       - name: Lint, build, and test
-        run: cd webonary-cloud-api; npm run lint && npm run test
+        run: cd webonary-cloud-api; npm run test
```

Comment:
`npm run test` does lint, compile, and test, so no need to run `lint` seperately